monitor/src/main/java/bisq/monitor/Configurable.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.monitor;
+
+import java.util.Properties;
+
+/**
+ * Configurable is configurable.
+ * 
+ * @author Florian Reimair
+ */
```

Comment:
I'd drop any boilerplate Javadoc like this and those that follow. Fine to have it if it really explains something, but otherwise it's line noise.

Reply:
agreed. Updated. I fear that I have been too much in a hurry to get the thing up and running :)